app/helpers/application_helper/button/generic_feature_button_with_disable.rb

```diff
@@ -2,19 +2,20 @@ class ApplicationHelper::Button::GenericFeatureButtonWithDisable < ApplicationHe
   needs :@record
 
   def calculate_properties
-    self[:enabled] = !(self[:title] = @error_message if disabled?)
+    super
+    self[:title] = @error_message if disabled?
   end
 
   def disabled?
     begin
       begin
         @error_message = @record.try(:unsupported_reason, @feature)
-      rescue NoMethodError # TODO: remove with deleting AvailabilityMixin module
-        @error_message = @record.try(:is_available_now_error_message, @feature) if @error_message.nil?
+      rescue SupportsFeatureMixin::UnknownFeatureError # TODO: remove with deleting AvailabilityMixin module
+        @error_message = @record.try(:is_available_now_error_message, @feature)
       end
     rescue
       @error_message = 'Feature is not supported.'
-      true
     end
+    return true if @error_message.present?
```

Comment:
why not make this return true/false only to make it a true predicate method.  The way to do that is to change the line to the simpler:
```ruby
return @error_message.present?
```
or the more idiomatic
```ruby
@error_message.present?
```

This will return true/false.